lib/livereload.js

```diff
@@ -112,7 +113,17 @@ function StartServer() {
             if (err) {
                 deferred.reject(err);
             }
+
             var server_url = bs.options.getIn(['urls', 'external']);
+
+            // In case there is no external url
+            // e.g: When the machine is not connected to any network
+            if (!server_url) {
+		// Usually, this err msg ends up being null in this case.
+		var error = new Error('No External URLs available. Make sure your computer is connected to a network.' + (err ? err.msg : ''));
```

Comment:
Handled with this commit: https://github.com/omefire/cordova-plugin-livereload/commit/c7c9ae64907ead75f9860bf4112bf7f867119346